### PR TITLE
Docs: Column resize and reorder

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: components/gantt/gantt-tree/columns/reorder.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+tags: telerik,blazor,gantt,column,reorder,drag
+published: True
+position: 2
+---
+
+# Reorder Columns
+
+The Gantt Tree lets the user reorder its columns by dragging their headers.
+
+To enable the column reordering, set the `Reorderable` parameter of the respective `GanttColumn` to `true`.
```
Comment (dimodi) [inline components/gantt/gantt-tree/columns/reorder.md:15]:
```suggestion
To enable column reordering, set the `Reorderable` parameter of the respective `GanttColumn` to `true`.
```

File: components/gantt/gantt-tree/columns/reorder.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+description: Drag to reorder columns in the Gantt Tree for Blazor.
+slug: gantt-columns-reorder
+tags: telerik,blazor,gantt,column,reorder,drag
+published: True
+position: 2
+---
+
+# Reorder Columns
+
+The Gantt Tree lets the user reorder its columns by dragging their headers.
```
Comment (dimodi) [inline components/gantt/gantt-tree/columns/reorder.md:13]:
```suggestion
The Gantt Tree lets the user reorder columns by dragging their headers.
```

File: components/gantt/gantt-tree/columns/resize.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+description: Drag to resize columns in the treelist for Blazor.
+slug: gantt-columns-resize
+tags: telerik,blazor,gantt,column,resize,drag
+published: True
+position: 3
+---
+
+# Resize Columns
+
+The Gantt Tree lets the user resize its columns by dragging the borders between their headers.
```
Comment (dimodi) [inline components/gantt/gantt-tree/columns/resize.md:13]:
```suggestion
The Gantt Tree lets the user resize columns by dragging the borders between their headers.
```

File: components/gantt/gantt-tree/columns/resize.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+To enable the column resizing, set the `Resizable` parameter of the `GanttColumn` to `true`.
```
Comment (dimodi) [inline components/gantt/gantt-tree/columns/resize.md:15]:
```suggestion
To enable column resizing, set the `Resizable` parameter of the `GanttColumn` to `true`.
```

File: components/gantt/gantt-tree/columns/resize.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+To prevent the user from resizing a certain column, set its own parameter `Resizable="false"`. Note that the user can still resize other columns around it.
```
Comment (dimodi) [inline components/gantt/gantt-tree/columns/resize.md:17]:
```suggestion
To disable resizing of a certain column, set its own parameter `Resizable="false"`. Note that the user can still resize other columns around it.
```

File: components/gantt/gantt-tree/columns/resize.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+When column resizing is enabled, a double click on the resize handle between the header cells will automatically fit the column width to the content of the header and data.
```
Comment (dimodi) [inline components/gantt/gantt-tree/columns/resize.md:19]:
```suggestion
When column resizing is enabled, a double click on the resize handle between two header cells will automatically fit the column width to the content of the header and data.
```

**Review by dimodi — APPROVED** [review]

File: components/gantt/gantt-tree/columns/resize.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
+---
+title: Resize
+page_title: TreeList - Resize Columns
+description: Drag to resize columns in the treelist for Blazor.
```
Comment (marin-bratanov) [inline components/gantt/gantt-tree/columns/resize.md:4]: wrong component name in title and description

Comment (marin-bratanov) [reply components/gantt/gantt-tree/columns/resize.md:4]: I do not need to see this again for second approval

Comment (svdimitr) [reply components/gantt/gantt-tree/columns/resize.md]: Done, thank you for the feedback

**Pull request merged**